migrations/sql/V1.21__Alter_Mine_Location_Add_Geoms.sql

```diff
@@ -0,0 +1,5 @@
+-- SRID=3005 is ‘BC Albers’ projection, chosen as the standard projection for this application.
+ALTER TABLE mine_location ADD COLUMN geom GEOMETRY(POINT, 3005);
+UPDATE mine_location SET geom = ST_SetSRID(ST_MakePoint(longitude, latitude),3005) WHERE longitude IS NOT NULL AND latitude IS NOT NULL;
+CREATE INDEX mine_location_geom_idx ON mine_location USING GIST(geom);
+--VACUUM ANALYZE mine_location;
```

Comment:
Unused comment?

Reply:
Removed the VACUUM comment.  The 3005 one ads value so leaving it.